### ISSUE TITLE
fatal: Not a valid object name develop

### ISSUE THREAD

**Issue by dvdklnr** [rhyeal/aws-rotate-iam-keys#40]
Details

` brew tap rhyeal/aws-rotate-iam-keys https://github.com/rhyeal/aws-rotate-iam-keys

==> Tapping rhyeal/aws-rotate-iam-keys
Cloning into '/usr/local/Homebrew/Library/Taps/rhyeal/homebrew-aws-rotate-iam-keys'...
remote: Enumerating objects: 56, done.
remote: Counting objects: 100% (56/56), done.
remote: Compressing objects: 100% (39/39), done.
remote: Total 1035 (delta 22), reused 39 (delta 12), pack-reused 979
Receiving objects: 100% (1035/1035), 564.90 KiB | 3.42 MiB/s, done.
Resolving deltas: 100% (507/507), done.
fatal: Not a valid object name develop
Tapped 1 formula (79 files, 924KB).
➜  Downloads brew upgrade
fatal: Not a valid object name develop`

**Comment by dvdklnr** [comment]
`/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::TapLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tbb.rb
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::TapLoader): loading /usr/local/Homebrew/Library/Taps/rhyeal/homebrew-aws-rotate-iam-keys/Formula/aws-rotate-iam-keys.rb
fatal: Not a valid object name develop
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::TapLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/awscli.rb
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::TapLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/geoip.rb`

**Comment by tom71** [comment]
`./aws-rotate-iam-keys.template.rb:      url %x{git config --local --get remote.origin.url | tr -d '\n'}, using: :git, branch: "develop"`
`./Formula/aws-rotate-iam-keys.rb:      url %x{git config --local --get remote.origin.url | tr -d '\n'}, using: :git, branch: "develop"
`

There is no branch called 'develop'.

**Comment by pepastach** [comment]
```
Warning: Calling 'devel' blocks in formulae is deprecated! Use 'head' blocks or @-versioned formulae instead.
Please report this issue to the rhyeal/aws-rotate-iam-keys tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/rhyeal/homebrew-aws-rotate-iam-keys/Formula/aws-rotate-iam-keys.rb:21

fatal: Not a valid object name develop
```

**Comment by mmrwoods** [comment]
@dvdklnr @tom71 @pepastach @veriservus The "fatal: Not a valid object name develop" issue has finally been fixed in version 0.9.8.1.

Sorry about the long delay in addressing this. Please run `brew update` to get the latest version of the formula.

The deprecation warning is a separate issue which we'll sort out soon.
